src/components/Dapp.re

```diff
@@ -539,7 +547,8 @@ module AnimalInfoStats = {
       QlHooks.useRemainingDepositEth(currentPatron)
       ->mapd(("Loading", "Loading"), a =>
           (
-            a->Eth.get(Eth.Eth(`ether)),
+            (a->Eth.get(Eth.Eth(`ether))->Float.fromString |||| 0.0)
+            ->toFixedWithPrecisionNoTrailingZeros(~digits=9),
```

Comment:
These numbers used to have waaay to many digits. I feel 9 is a happy middle, it isn't crazy long, but you can see the numbers ticking up.